tests/test_presolve.py

```diff
@@ -247,3 +247,22 @@ def test_variable_removal(self):
         presolver = Presolver(cqm)
         presolver.load_default_presolvers()
         presolver.apply()
+
+    def test_zero_biases(self):
+        i, j = dimod.Integers('ij')
+        cqm = dimod.ConstrainedQuadraticModel()
+
+        cqm.add_constraint(0*i >= -5)
```

Comment:
and here, unless it is already covered, should we test that if `0*i >= num` where `num>0` raises the infeasible error? Unless it's already covered.

Reply:
There is another branch kicking around with more of that, but it's not ready for merge yet.

Reply:
sounds good!